metron-platform/metron-parsing/README.md

```diff
@@ -624,6 +624,8 @@ Java parser adapters are intended for higher-velocity topologies and are not eas
 * org.apache.metron.parsers.lancope.BasicLancopeParser : Parse Lancope messages
 * org.apache.metron.parsers.syslog.Syslog5424Parser : Parse Syslog RFC 5424 messages
 * org.apache.metron.parsers.syslog.Syslog3164Parser : Parse Syslog RFC 3164 messages
+* org.apache.metron.parsers.cef.CEFParser: Parse CEF format messages
```

Comment:
@ottobackwards @simonellistonball - Do we not have any documentation for configuring any of the Java parsers? Not really of consequence for the LEEF parser since the config method is empty, but what about the others? e.g. `BaseSyslogParser` appears to have some config around device timezone, but there's no mention of it in our README.

Reply:
Not really at the moment. We could clean that up, but in this case, LEEF is leaf. As you say, in this case we don't really have anything to document, since there is no config.